src/Fable.ReactGoogleMaps/Fable.Helpers.ReactGoogleMaps.fs

```diff
@@ -0,0 +1,172 @@
+module Fable.Helpers.ReactGoogleMaps
+
+open Fable.Core
+open Fable.Import
+open Fable.Core.JsInterop
+
+module Coordinates =
+
+    type [<Pojo>] Position = {
+        lat: float
+        lng: float
+    }
+
+    let newPos lat lng =
+        { lat = lat
+          lng = lng }
+
+    type [<Pojo>] Bounds = {
+        NE : Position
+        SW : Position
+    }
+
+
+module Places =
+
+    type [<Pojo>] Geometry = {
+        location: Coordinates.Position
+    }
+
+    type [<Pojo>] Place = {
+        geometry: Geometry
+    }
+
+
+module R = Fable.Helpers.React
+
+type RCom = React.ComponentClass<obj>
+
+type MapRef(mapRef) =
+
+    /// Get the current bounds of the Map
+    member __.GetBounds() : Coordinates.Bounds =
+        let bounds = mapRef?getBounds()
+        let ne = bounds?getNorthEast()
+        let sw = bounds?getSouthWest()
+
+        { NE = Coordinates.newPos (ne?lat() |> unbox) (ne?lng() |> unbox)
+          SW = Coordinates.newPos (sw?lat() |> unbox) (sw?lng() |> unbox) }
+
+    member __.GetZoom() : int =
+        mapRef?getZoom() |> unbox
+
+    member __.GetCenter() : Coordinates.Position =
+        mapRef?getCenter() |> unbox
+
+module Props =
+
+
+    type IInfoWindowProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type InfoWindowProperties =
+    | OnCloseClick of (unit -> unit)
+        interface IInfoWindowProperties
+
+    type ISearchBoxProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type SearchBoxProperties =
+    | Ref of (obj -> unit)
+    | OnPlacesChanged of (unit -> unit)
+        interface ISearchBoxProperties
+
+    type IMarkerProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MarkerProperties =
+    | Key of obj
+    | Title of string
+    | Icon of string
+    | OnClick of (unit -> unit)
+    | Position of Coordinates.Position
+        interface IMarkerProperties
+
+    type IMarkerClustererProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MarkerClustererProperties =
+    | Key of obj
+    | AverageCenter of bool
+    | MaxZoom of int
+    | EnableRetinaIcons of bool
+    | GridSize of float
+        interface IMarkerClustererProperties
+
+    type IMapProperties =
+        interface end
+
+    [<RequireQualifiedAccess>]
+    type MapProperties =
+    | SetRef of (obj -> unit)
+    | ApiKey of string
+    | DefaultZoom of int
+    | SearchBoxText of string
+    | ShowSearchBox of bool
+    | ShowTrafficLayer of bool
+    | DefaultCenter of Coordinates.Position
+    | Center of Coordinates.Position
+    | OnCenterChanged of (unit -> unit)
+    | OnPlacesChanged of (Places.Place [] -> unit)
+    | OnZoomChanged of (unit -> unit)
+    | OnIdle of (unit -> unit)
+    | Markers of React.ReactElement
+    | MapLoadingContainer of string
+    | MapContainer of string
+        interface IMapProperties
+let InfoWindow: RCom = import "InfoWindow" "react-google-maps"
```

Comment:
can we use fable to add this dependency automatically? if not I guess it's fine I write it in readme

Reply:
No, atm there's no mechanism to enforce npm dependencies. I was hoping Paket would do that for us ;)